mtp_cashbook/apps/mtp_auth/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class ConnectionError(Exception):
+    """
+    Exception that should be raised in case of connection
+    problems with the api server.
+    """
+    pass
```

Comment:
Do we need our own `ConnectionError` class? Isn't this a little confusing, given that [Python 3 provides a `ConnectionError`](https://docs.python.org/3/library/exceptions.html#ConnectionError)? Should we just use that (or make this a subclass… or just pass out whatever the underlying error is)?